src/inputSources/mouseInput.js

```diff
@@ -108,11 +108,9 @@ function mouseDown (e) {
     type: eventType
   };
 
-  const event = external.$.Event(eventType, eventData);
+  const eventPropogated = triggerEvent(eventData.element, eventType, eventData);
```

Comment:
minor comment, but propagated is spelled wrong

Reply:
Fixed